tasks/download-badge-api-package.sh

```diff
@@ -2,7 +2,23 @@
 # After that, the package directory can be deployed to azure with package deploy
 set -e
 NPM_PACKAGE_VERSION=$1
-wget $(npm view @stryker-mutator/dashboard-badge-api@${NPM_PACKAGE_VERSION} dist.tarball)
+
```

Comment:
Did you change this file by mistake?

Reply:
You made these changes in c95ad181d629f73c45af79dad198ce4e06f25fb3 :) My commits do not touch these files.